src/client/pages/OfferNew/Introduction/Sidebar/Sidebar.stories.tsx

```diff
@@ -35,3 +39,47 @@ export const Defualt = () => {
     </MockedProvider>
   )
 }
+
+export const NOCombo = () => {
+  const ref = useRef<HTMLDivElement>(null)
+  return (
+    <MockedProvider>
+      <MemoryRouter initialEntries={['/no/new-member/offer']}>
+        <TextKeyProvider locale="nb_NO">
+          <Sidebar
+            ref={ref}
+            sticky={false}
+            offerData={noCombo}
+            onCheckoutOpen={action('onCheckoutOpen')}
+            refetch={(...args: any[]) => {
+              action('refetch')(...args)
+              return Promise.resolve()
+            }}
+          />
+        </TextKeyProvider>
+      </MemoryRouter>
+    </MockedProvider>
+  )
+}
+
+export const DKHomeContent = () => {
+  const ref = useRef<HTMLDivElement>(null)
+  return (
+    <MockedProvider>
+      <MemoryRouter initialEntries={['/no/new-member/offer']}>
+        <TextKeyProvider locale="nb_NO">
```

Comment:
Should be Dk? 🇩🇰  "nb_NO" and "no/new/member"

Reply:
oops, good catch! 👌